homeassistant/components/zone/trigger.py

```diff
@@ -67,16 +64,16 @@ async def async_attach_trigger(
     """Listen for state changes based on configuration."""
     trigger_data = trigger_info["trigger_data"]
     entity_id: list[str] = config[CONF_ENTITY_ID]
-    zone_entity_id = config.get(CONF_ZONE)
-    event = config.get(CONF_EVENT)
+    zone_entity_id = config[CONF_ZONE]
+    event = config[CONF_EVENT]
```

Comment:
`CONF_ZONE` and `CONF_EVENT` are compulsory, according to `_TRIGGER_SCHEMA`

Reply:
```suggestion
    zone_entity_id: str = config[CONF_ZONE]
    event: str = config[CONF_EVENT]
```
I usually recommend adding type annotations in these cases. Without it, mypy would infer them as `Any` (from `ConfigType`).